core/src/main/resources/reference.conf

```diff
@@ -206,183 +77,188 @@ cassandra-journal {
-  # The id for the local datacenter of the Cassandra hosts it should connect to.
-  # By default, this property is not set resulting in Datastax's standard round robin policy being used.
-  local-datacenter = ""
+    # Target number of entries per partition (= columns per row).
+    # Must not be changed after table creation (currently not checked).
+    # This is "target" as AtomicWrites that span partition boundaries will result in bigger partitions to ensure atomicity.
+    target-partition-size = 500000
 
-  # Number of hosts from non-local datacenter to use as a fall-back policy.
-  # Works only when local-datacenter is set
-  used-hosts-per-remote-dc = 0
+    # The time to wait before cassandra will remove the tombstones created for deleted entries.
+    # cfr. gc_grace_seconds table property documentation on
+    # http://www.datastax.com/documentation/cql/3.1/cql/cql_reference/tabProp.html
+    gc-grace-seconds = 864000
 
-  # Dispatcher for the plugin actor and task.
-  plugin-dispatcher = "cassandra-plugin-default-dispatcher"
+    # Akka-persistence allows multiple pending deletes for the same persistence id however this plugin only executes one
+    # at a time per persistence id (deletes for different persistenceIds can happen concurrently).
+    #
+    # If multiple deletes for the same persistence id are received then they are queued.
+    #
+    # If the queue is full any subsequent deletes are failed immediately without attempting them in Cassandra.
+    #
+    # Deleting should be used with snapshots for efficiency for recovery thus deleting should be infrequent
+    max-concurrent-deletes = 10
+
+    # For applications that are not deleting any events this can be set to 'off', which will optimize
+    # the recovery to not query for highest deleted sequence number from the metadata table.
+    # It must not be off if deletes of events are used or have been used previously.
+    # If this is set to off then delete attempts will fail with an IllegalArgumentException.
+    support-deletes = on
+  }
 
-  # Set the protocol version explicitly, should only be used for compatibility testing.
-  # Supported values: 3, 4
-  protocol-version = ""
+  # FIXME update docs
+  # This configures the default settings for all CassandraReadJournal plugin
+  # instances in the system.
+  #
+  # If you use multiple plugin instances you need to create differently named
+  # sections containing only those settings that shall be different from the defaults
+  # configured here, importing the defaults like so:
+  #
+  #   my-cassandra-query-journal = ${cassandra-query-journal}
+  #   my-cassandra-query-journal {
+  #     <settings...>
+  #   }
+  read {
+    # Implementation class of the Cassandra ReadJournalProvider
+    class = "akka.persistence.cassandra.query.CassandraReadJournalProvider"
+
+    # Dispatcher for the plugin actors.
+    plugin-dispatcher = "cassandra-plugin-default-dispatcher"
+
+    # FIXME #81 is this needed in read section?
```

Comment:
maybe these could be better named. The journal does writes and reads to cassandra so it has two profiles.

The read side only does reads and can have a separate read profile to the reading done in the journal